src/core/fonts.js

```diff
@@ -2901,13 +2901,18 @@ var Font = (function FontClosure() {
         }
 
         // Last, try to map any missing charcodes using the post table.
-        if (properties.glyphNames && baseEncoding.length) {
+        if (
+          properties.glyphNames &&
+          (baseEncoding.length || this.differences.length)
+        ) {
           for (let i = 0; i < 256; ++i) {
-            if (charCodeToGlyphId[i] === undefined && baseEncoding[i]) {
-              glyphName = baseEncoding[i];
-              const glyphId = properties.glyphNames.indexOf(glyphName);
-              if (glyphId > 0 && hasGlyph(glyphId)) {
-                charCodeToGlyphId[i] = glyphId;
+            if (charCodeToGlyphId[i] === undefined) {
+              glyphName = this.differences[i] || baseEncoding[i];
+              if (glyphName) {
+                const glyphId = properties.glyphNames.indexOf(glyphName);
+                if (glyphId > 0 && hasGlyph(glyphId)) {
+                  charCodeToGlyphId[i] = glyphId;
+                }
```

Comment:
*Optional nit:* I'd be sort-of inclined to reduce the overall indentation level in this loop, by using `continue`, e.g. something like:
```javascript
            if (charCodeToGlyphId[i] !== undefined) {
              continue;
            }
            glyphName = this.differences[i] || baseEncoding[i];
            if (!glyphName) {
              continue;
            }
            const glyphId = properties.glyphNames.indexOf(glyphName);
            if (glyphId > 0 && hasGlyph(glyphId)) {
              charCodeToGlyphId[i] = glyphId;
            }
```